go/kbfs/libkbfs/subscription_manager_test.go

```diff
@@ -0,0 +1,91 @@
+package libkbfs
+
+import (
+	"testing"
+	"time"
+
+	gomock "github.com/golang/mock/gomock"
+	"github.com/keybase/client/go/kbfs/data"
+	"github.com/keybase/client/go/kbfs/favorites"
+	"github.com/keybase/client/go/kbfs/tlf"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func delayedFinish(ctl *gomock.Controller, timeout time.Duration) (
+	finish func(), done func(args ...interface{})) {
+	ch := make(chan struct{})
+	return func() {
+			select {
+			case <-time.After(timeout):
```

Comment:
Should this fail the test if it triggers?  Or is there a legitimate reason this could fire?

Reply:
The mock tests would fail because it never saw the expect call. But I guess failing here is better. Will do!